universal-login-sdk/lib/core/services/TransferService.ts

```diff
@@ -16,13 +21,34 @@ export class TransferService {
   }
 
   validateInputs(transferDetails: TransferDetails, balance: Nullable<string>) {
+    this.errors = {amount: [], to: []};
     const {gasPrice, gasToken} = transferDetails.gasParameters;
     const gasCostInWei = utils.bigNumberify(DEFAULT_GAS_LIMIT.toString()).mul(gasPrice);
     ensureNotNull(balance, Error, 'Balance is null');
     const isAmountValid = isValidAmount(transferDetails.transferToken, balance, gasToken, gasCostInWei, transferDetails.amount);
+    if (!isAmountValid) {
+      this.errors.amount.push(`Amount ${transferDetails.amount} is not valid`);
+    }
     const isRecipientValid = isValidRecipient(transferDetails.to);
```

Comment:
inline isAmountValid and  isRecipientValid. I think we don't loose code readability:
```ts
if (!isValidAmount(.....)) {

}
```